plugins/module_utils/terraform_commands.py

```diff
@@ -240,6 +240,7 @@ def workspace_list(self) -> TerraformWorkspaceContext:
                 continue
             elif stripped_item.startswith("* "):
                 current_workspace = stripped_item.replace("* ", "")
+                all_workspaces.append(current_workspace)
```

Comment:
This change is unrelated to the PR, but I noticed that the `current_workspace` doesn't get added to `all_workspaces`.

Reply:
Thanks for catching this. It would be nice to at least have a unit test for this.